src/Components/Address/useAddressAutocomplete.ts

```diff
@@ -54,6 +57,32 @@ export const useAddressAutocomplete = (
     }
   }, [isUSAddress, result.length])
 
+  // clear address line 2 if selecting a new suggested address
+  useEffect(() => {
+    const newOptions = result.map(
+      (suggestion: ProviderSuggestion): AddressAutocompleteSuggestion => {
+        const text = buildAddressText(suggestion)
+        let newAddressLine2 = suggestion.secondary ? suggestion.secondary : ""
```

Comment:
This can just be suggestion.secondary || ""

Reply:
Agreed, but I am also exploring removing the useEffect() from this code and seeing how that works out.